### ISSUE TITLE
✨ Container Actor

### ISSUE THREAD

**Issue by moinen** [fvtt-fria-ligan/forbidden-lands-foundry-vtt#220]
**Is your feature request related to a problem? Please describe.**
One of that quality of life things. It would be great to have an option to unload stuff without creating a Stronghold or Character sheet.

**Describe the solution you'd like**
Create Actor with only name, description, and capacity fields. And item list obviously.

**Additional context**
It could be used as loot chest or a merchant's goods list. I imagine there would be many applications

**Comment by moinen** [comment]
It could be also used as a vehicle I think

**Comment by narukaioh** [comment]
This looks more like a module than a system feature. It's a great idea!
When we try to implement things for the system, we usually think about things related to the rules of the system.

**Comment by moinen** [comment]
You are totally right. It is not an essential functionality of the module but it would improve the experience if you could have Floor or Cart actor. 
And I think it would be fairly simple to do since it would be a stripped down version of the sheet
